js/README.md

```diff
@@ -17,50 +17,165 @@
   under the License.
 -->
 
-### Installation
+# [Apache Arrow](https://github.com/apache/arrow) in JS
 
-From this directory, run:
+[![Build Status](https://travis-ci.org/graphistry/arrow.svg?branch=master)](https://travis-ci.org/graphistry/arrow)
+[![Coverage Status](https://coveralls.io/repos/github/graphistry/arrow/badge.svg)](https://coveralls.io/github/graphistry/arrow)
 
-``` bash
-$ npm install   # pull dependencies
-$ npm run lint -- <filename>  # run tslint
-$ npm run build # build typescript (run tsc and webpack)
-$ npm run test  # run the unit tests (node.js only)
+Loading big native dataframes in JavaScript is finally awesome. `apache-arrow` provides an easy, modern, and efficient zero-copy JS interface to parse, iterate, and access [Apache Arrow](https://github.com/apache/arrow) columnar data on CPUs (GPU support via [GoAI](http://gpuopenanalytics.com/) is occurring in parallel).
```

Comment:
Update this copy to be more objective. Maybe remove reference to GoAI since there is work happening for GPUs in this codebase, too, so possible to give the mistaken impression that Apache Arrow is only concerned with CPUs